<?xml version="1.0"?>
<launch>
  <!--rosparam command="load" file="$(env HOME)/params.yaml" /-->

  <node pkg="interactive_goal" name="interactive_posestamped_node" type="interactive_posestamped_node" respawn="true"/>

<!-- set reg-->

    <node pkg="drone_reg" name="robot_reg" type="drone_reg_cpp" respawn="true" output="screen">
    <param name="use_planner" value="false"/>
    <param name="use_geo_mode" value="false"/>
    <param name="yaml_path" value="$(find robot_bringup)/configuration_files/pid_params.yaml"/>
    <param name="use_alt_sonar" value="false"/>
    <remap from="/mavros/local_position/pose"  to="/robot/local_position" />
    <remap from="/mavros/setpoint_velocity/cmd_vel"  to="robot/cmd_vel_stamp" />
</node>

</launch>
